src/tests/dune

```diff
@@ -1,6 +1,7 @@
-(library
+(test
  (name snarky_tests)
- (inline_tests)
- (libraries snarky)
+ (flags
+  (:standard -w -3))
```

Comment:
Do you need this?

Reply:
This was to get around some warning about deprecated `bin_io` related function that I couldn't figure out how to turn off otherwise

Reply:
Oh no :(